src/pio/mod.rs

```diff
@@ -1,28 +1,21 @@
-use std::{
```

Comment:
Isn't `mod.rs` old-skool now? Isn't just `pio.rs` preferred now, and if you have sub-modules, these just go in `/pio/...`?